addon/components/frost-fixed-table.js

```diff
@@ -143,21 +143,36 @@ export default Component.extend({
   },
 
   @readOnly
-  @computed('indexedColumns')
+  @computed('columns')
+  /**
+   * Pre-computed indices
+   * @param {Column[]} columns - the column data we want to present
+   * @returns {Boolean} true if the table will have categories, false otherwise
+   */
+  haveCategories (columns) {
+    return columns.reduce((cur, column) => { return cur || !isNone(column.category) }, false)
+  },
+
+  @readOnly
+  @computed('indexedColumns', 'haveCategories')
   /**
    * Get the set of columns that are supposed to be frozen on the left
    *
    * The set of leftColumns is defined as all the columns with `frozen` === `true`
    * starting with the first column until we reach one w/o `frozen` === `true`
    *
    * @param {Column[]} columns - all the columns
+   * @param {Boolean} haveCategories - if the header has categories
    * @returns {Column[]} just the left-most frozen columns
    */
-  leftColumns (columns) {
+  leftColumns (columns, haveCategories) {
     const frozenColumns = []
     for (let i = 0; i < columns.length; i++) {
       const column = columns[i]
       if (column.frozen) {
+        if (haveCategories && isNone(column.category)) {
```

Comment:
![screen shot 2017-08-17 at 8 55 37 am](https://user-images.githubusercontent.com/22083831/29457278-845ebffe-83e7-11e7-995c-21e104786b13.png)

This picture shows the problem where `Universe` is the column header not the category